src/firebolt_ingest/table_service.py

```diff
@@ -124,7 +123,9 @@ def insert_full_overwrite(
         """
         cursor = self.connection.cursor()
 
-        raise_on_tables_non_compatibility(cursor, self.table, ignore_meta_columns=True)
+        # raise_on_tables_non_compatibility(cursor,
```

Comment:
Please add a TODO comment to uncomment it back, also mention jira ticket

Reply:
done